steampipe.yaml

```diff
@@ -0,0 +1,93 @@
+package:
+  name: steampipe
+  version: 1.1.0
+  epoch: 0
+  description: Steampipe is the zero-ETL way to query APIs and services, used to expose data sources to SQL.
+  dependencies:
+    runtime:
+      - bash
+      - ca-certificates-bundle
+      - gcc
```

Comment:
gcc is not required as a runtime dependency here.

Reply:
steampipe installs and sets up postges, fdw as a part of initial setup.